### PR TITLE
Improvement/kudos sending ux

### PR DESCRIPTION
**Pull request by edsdck**
New modal for sending kudos to other users. 

kudossending

### CODE REVIEW

File: src/api/Main/PresentationLayer/Shrooms.API/Controllers/Kudos/KudosController.cs [hunk 1]
```diff
@@ -150,6 +151,14 @@ public IEnumerable<KudosTypeViewModel> GetKudosTypes()
             return kudosTypeViewModel;
         }
 
+        [HttpGet]
+        [PermissionAuthorize(Permission = BasicPermissions.Kudos)]
+        public KudosTypeViewModel GetSendType()
```
Comment (Aleksanderis) [inline src/api/Main/PresentationLayer/Shrooms.API/Controllers/Kudos/KudosController.cs]: Please keep consistent naming. If it's `_kudosService.GetKudosTypeSend`, then why controller endpoint named in different pattern? Also if it's more natural to say `GetSendType`, then perhaps in service it also could be named - `GetSendKudosType`, and then controller endpoint accordingly.

File: src/webapp/src/client/resources/en_US/kudos.json [hunk 2]
```diff
@@ -20,7 +20,7 @@
   "kudosType": "Type",
   "multiplier": "Multiplier",
   "points": "Points",
-  "kudos.amountOfKudos": "Amount of Kudos",
```
Comment (Aleksanderis) [inline src/webapp/src/client/resources/en_US/kudos.json:23]: Shouldn't it be changed in other language files as well?

File: src/webapp/src/client/app/kudos/wall-widget/wall-widget.html [hunk 3]
```diff
@@ -2,16 +2,16 @@
...
-            <a ace-kudosify-modal ga-track-event="['Kudos', 'wall big plus', 'opened']" data-test-id="kudosify-button-plus">
+            <a ace-kudosify-modal kudosify-type="{{vm.modalTypes.submit}}" ga-track-event="['Kudos', 'wall big plus', 'opened']" data-test-id="kudosify-button-plus">
                 <span uib-tooltip="{{'kudos.submitKudos' | translate}}" 
                       tooltip-placement="bottom" 
                       tooltip-class="word-break" 
                       class="glyphicon glyphicon-plus kudosify-big-plus-btn" 
                       aria-hidden="true"></span>
             </a>
         </div>
-        <div class="kudos-big-button" feature-flag="kudosSendImproved">
```
Comment (Aleksanderis) [inline src/webapp/src/client/app/kudos/wall-widget/wall-widget.html:13]: If "kudosSendImproved" feature-flag no longer needed, then it should be also removed from `config.js`

**Review by Aleksanderis — CHANGES_REQUESTED** [review]

File: src/webapp/src/client/resources/en_US/kudos.json (hunk 2 above)
Comment (Kristupas13) [reply src/webapp/src/client/resources/en_US/kudos.json:23]: It already existed in en_US language
image
So just fixed the naming. (It had no references)

Other file already had this translation:
image

**Review by Aleksanderis — APPROVED** [review]

**Pull request merged**